sxr-x3d-demo/app/build.gradle

```diff
@@ -33,9 +32,8 @@ dependencies {
     }
     implementation 'com.google.android.exoplayer:exoplayer:2.6.0'
     // ARCore library (copied from gvr-arcore
-    implementation "com.google.ar:core:1.2.0"
+    implementation 'com.google.ar:core:1.2.0'
```

Comment:
I believe you can use the same ARCore version that is being used in MixedReality extension (see https://github.com/sxrsdk/sxrsdk/blob/master/SXR/Extensions/MixedReality/build.gradle#L36)

Reply:
Perhaps, I am no expert in Build.gradle files.  The issue would be that X3D demo's must also support playing videos with EXoplayer.  Would using the same build.gradle as MixedReality still support that?

Reply:
No, you need to put ExoPlayer as explicit dependency (it is ok in your gradle file).

We've tried to put ARCore in MixedReality package to make the applications use only MixedReality as explicit dependency but it was not possible unfortunately (see https://github.com/sxrsdk/sxrsdk-demos/pull/64).